acl/lang/en

```diff
@@ -519,12 +519,10 @@ twofactor_ebutton=No button clicked!
 forgot_title=Send Password Reset Link
 forgot_err=Failed to send password reset link
 forgot_header=Password reset link details
-forgot_user=Reset password for Webmin user
+forgot_user=Reset password for Webmin or sudo-capable Unix user
```

Comment:
I don't like that it mentions "sudo-capable Unix user" here, as this message is misleading for the common case where only a Webmin password is being reset. Instead just make it "Reset password for user"

Reply:
Agreed! Fixed now!